CreateTablesScript.sql

```diff
@@ -0,0 +1,118 @@
+CREATE TABLE IF NOT EXISTS `mydb`.`USERS` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `email` VARCHAR(45) NOT NULL,
+  `username` VARCHAR(45) NOT NULL,
+  `password` VARCHAR(45) NOT NULL,
+  `gender` INT NOT NULL,
+  `birthdate` DATE NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `id_UNIQUE` (`id` ASC) VISIBLE,
+  UNIQUE INDEX `password_UNIQUE` (`password` ASC) VISIBLE,
+  UNIQUE INDEX `login_UNIQUE` (`username` ASC) VISIBLE)
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`PARAMETERS`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`PARAMETERS` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `user_id` INT NOT NULL,
+  `weight` FLOAT NOT NULL,
+  `height` FLOAT NOT NULL,
+  `physical_acitvity` INT NOT NULL,
+  `target_weight` FLOAT NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `id_UNIQUE` (`id` ASC) VISIBLE,
+  UNIQUE INDEX `user_id_UNIQUE` (`user_id` ASC) VISIBLE,
+  CONSTRAINT `user_id_fk`
+    FOREIGN KEY (`user_id`)
+    REFERENCES `mydb`.`USERS` (`id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`INGREDIENTS`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`INGREDIENTS` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `calories` INT NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `id_UNIQUE` (`id` ASC) VISIBLE)
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`DISHES`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`DISHES` (
```

Comment:
Mb we can add also a photo? We can create photo_url column to have a link to some image storage.